### PR TITLE
Update streaming install docs for KEDA on k8s v1.16 or later

### LINKED ISSUE #1377: Need to solve keda install for k8s v1.16+
We can't use `kapp` for installing `keda` on Kubernetes v1.16 or later since `keda` still use `k8s.io` in the API group.

- message: protected groups must have approval annotation "api-approved.kubernetes.io",
   see https://github.com/kubernetes/enhancements/pull/1111

### PR DESCRIPTION
**Pull request by trisberg**
Fixes projectriff/riff/issues/1377

### CODE REVIEW

File: docs/v0.5/runtimes/streaming.md [hunk 1]
```diff
@@ -6,22 +6,36 @@ sidebar_label: Streaming
...
+
 ## Install
 
 The Streaming runtime is not installed with riff by default. See the [getting started guide](../getting-started.md) for how to install the prerequisits and riff Build in your Kubernetes environment.
 
 You can then install the Streaming runtime and it's dependencies using the following:
 
+### install KEDA
+
+> NOTE: For Kubernetes versions 1.16 and later the use of `k8s.io` in the KEDA CRDs results in a [failed status condition](https://github.com/kedacore/keda/issues/552) and we need to avoid the kapp reconciliation for the KEDA resources.
```
Comment (scothis) [inline docs/v0.5/runtimes/streaming.md:19]: rm let's fix this in the install

**Review by scothis — CHANGES_REQUESTED** [review]

**Review by jldec — APPROVED** [review]
LGTM

(curious: what would be the risk/harm of using `wait=false` for the keda install on all k8s versions?)

**Comment by trisberg** [comment]
I think using `--wait=false` for all k8s versions would be OK as well, the risk is that keda doesn't start up for some other reason

File: docs/v0.5/runtimes/streaming.md (hunk 1 above)
Comment (trisberg) [reply docs/v0.5/runtimes/streaming.md:19]: what do you mean by "fix this in the install"?

Comment (scothis) [reply docs/v0.5/runtimes/streaming.md:19]: https://github.com/projectriff/release/pull/129

**Comment by trisberg** [comment]
Closed in favor of https://github.com/projectriff/release/pull/129

**Pull request closed without merging**